airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/debezium/internals/DebeziumRecordIterator.kt

```diff
@@ -246,5 +265,13 @@ class DebeziumRecordIterator<T>(
         }
     }
 
+    enum class DebeziumCloseReason() {
```

Comment:
I guess this would also be part of our efforts on propagating DBZ errors related to [this Q2B epic](https://github.com/airbytehq/airbyte/issues/39570), correct?